### PR TITLE
Fix: Failed to verify, file not found.

### PR DESCRIPTION
**Pull request by maaktweluit**
- Use `get_expected_extension( ` for result files
- Added logging for collecting results
- Updated pillow like in golem-core

### CODE REVIEW

File: image/golem_blender_app/requirements.txt [hunk 1]
```diff
@@ -5,7 +5,7 @@ numpy==1.15.4
 opencv-python==4.0.0.21
 OpenEXR==1.3.2
 peewee==3.11.2
-Pillow==5.1.0
+Pillow==6.2.0
```
Comment (shadeofblue) [inline image/golem_blender_app/requirements.txt]: @maaktweluit have you confirmed the image works correctly with the upgraded version of Pillow?

Comment (maaktweluit) [reply image/golem_blender_app/requirements.txt]: this change was taken from golem-core https://github.com/golemfactory/golem/pull/4875

I have not checked if the image works, i assumed the CGI team had already done this.

File: image/golem_blender_app/golem_blender_app/commands/verify.py [hunk 2]
```diff
@@ -104,16 +105,23 @@ def _collect_results(
...
         height=params['resolution'][1],
     )
     for i in subtasks_nums[::-1]:
-        path = work_dir / f'subtask{subtasks_statuses[i][1]}' / 'results' / \
-            f'result{frame:04d}.{out_format}'
-        collector.add_img_file(str(path))
+        result_dir = work_dir / f'subtask{subtasks_statuses[i][1]}' / 'results'
+        result_img = result_dir / f'result{frame:04d}.{out_format}'
+        print(f'result_dir: {result_dir}')
+        for file in result_dir.iterdir():
```
Comment (mfranciszkiewicz) [inline image/golem_blender_app/golem_blender_app/commands/verify.py]: This name shadows the `file` is a builtin function

Comment (maaktweluit) [reply image/golem_blender_app/golem_blender_app/commands/verify.py]: good catch, will update

**Pull request merged**